JsBasics/Conditions_and_loops/student_grades.js

```diff
@@ -0,0 +1,8 @@
+let averagegrade = 85
```

Comment:
Задля читабельності коду:
```suggestion
let averageGrade = 85
```